### ISSUE TITLE
(MzIdentMLWriter) SpectrumIdentification.write TypeError missing positional argument xml_file

### ISSUE THREAD

**Issue by lars-kolbowski** [mobiusklein/psims#20]
Hi Joshua,
another error I ran into when following the [example](https://mobiusklein.github.io/psims/docs/build/html/mzid/examples.html).

When running the AnalysisCollection SpectrumIdentification(...).write I get a TypeError:
```
with writer.analysis_collection():
    writer.SpectrumIdentification(spectra_data_ids_used=[
        "mgf_1", "mgf_2", "mgf_3"
    ], search_database_ids_used=[
        "search_db_1"
    ], spectrum_identification_list_id='spectra_identified_list_1',
        spectrum_identification_protocol_id='spectrum_identification_params_1').write()
```

Throws: 
`spectrum_identification_protocol_id='spectrum_identification_params_1').write()
TypeError: ComponentBase.write() missing 1 required positional argument: 'xml_file'`

**Comment by mobiusklein** [comment]
Thank you for reporting. This was written with an assumption of a feature that wasn't finished being implemented.

**Comment by mobiusklein** [comment]
This should be fixed in v1.2.6 which is being released to PyPI momentarily.

**Comment by lars-kolbowski** [comment]
Thank you!
